code/modules/antagonists/heretic/knowledge/blade_lore.dm

```diff
@@ -444,7 +444,7 @@
 	)
 	ADD_TRAIT(user, TRAIT_NEVER_WOUNDED, name)
 	RegisterSignal(user, COMSIG_HERETIC_BLADE_ATTACK, PROC_REF(on_eldritch_blade))
-	user.apply_status_effect(/datum/status_effect/protective_blades/recharging, null, 8, 30, 0.25 SECONDS, 1 MINUTES)
+	user.apply_status_effect(/datum/status_effect/protective_blades/recharging, null, 8, 30, 0.25 SECONDS, 1 MINUTES, /obj/projectile/floating_blade)
```

Comment:
I am pretty certain you should not be passing a projectile to spawn in place of an `/obj/effect`

Reply:
It always has been